PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# Description
```

Comment:
This file should typically be present in the .github directory or the docs/ directory.